test/components/routers/test_conditional_router.py

```diff
@@ -574,3 +574,34 @@ def test_router_to_dict_does_not_mutate_routes(self):
         assert new_router.routes == router.routes
         assert new_router.routes[0]["output_type"] is str
         assert new_router.routes[0]["output_type"] is original_output_type
+
+    def test_multiple_outputs_per_route(self):
+        """Test that router handles multiple outputs per route correctly"""
+        routes = [
+            {
+                "condition": "{{streams|length >= 2}}",
+                "output": ["{{streams}}", "{{query}}"],
+                "output_type": [List[int], str],
+                "output_name": ["streams", "query"],
+            }
+        ]
```

Comment:
Only one minor comment. Could we expand the routes here to include two routes instead of just one?